modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceShipments.testcase

```diff
@@ -109,12 +109,78 @@ definition {
 	}
 
 	@description = "This is a test for COMMERCE-6250. Delete shipment"
-	@ignore = "true"
 	@priority = "3"
 	test DeleteShipment {
 		property portal.acceptance = "false";
 
-		// TODO COMMERCE-6250 DeleteShipment Pending Implementation
+		CommerceAccelerators.initializeNewSiteViaAccelerator(siteName = "Minium");
+
+		ApplicationsMenu.gotoSite(site = "Minium");
+
+		CommerceNavigator.gotoMiniumSidebarItem(menuItem = "Account Management");
+
+		CommerceAcceleratorsInitializer.createNewAccountViaFront(
+			accountName = "Commerce Account",
+			adminEmailAddress = "test@liferay.com");
+
+		ApplicationsMenu.gotoSite(site = "Minium");
+
+		FrontStore.addInitializerProductToCart(
+			productName = "U-Joint",
+			productQuantity = "1");
+
+		CommerceNavigator.gotoMiniCart();
+
+		Button.click(button = "Submit");
```

Comment:
Because of a UI issue occurring on CI, plaese substitute this step with:

```
Button.click(button = "View Details");

		Button.click(button = "Checkout");

		Pause(locator1 = "3000");
```

Reply:
Only
```
Button.click(button = "Submit");
```
needs to be removed and substituted.

Reply:
Ok, done!